srcCpp/RTIDDSImpl.cxx

```diff
@@ -2126,24 +2127,17 @@ double RTIDDSImpl<T>::obtain_dds_serialize_time_cost(
     }
 
     /* --- Initialize data --- */
-    data.bin_data.maximum(0);
     data.entity_id = 0;
     data.seq_num = 0;
     data.timestamp_sec = 0;
     data.timestamp_usec = 0;
     data.latency_ping = 0;
-
-    /* Does not matter what it's inside */
-    data.bin_data.loan_contiguous(
```

Comment:
File a bug about this.

Reply:
It is not possible to deserialize a type containing an unbounded seq using a loaned buffer with enough size. This was not happening in 5.3.1